pkg/apis/uiplugin/v1alpha1/types.go

```diff
@@ -167,9 +171,23 @@ type AlertmanagerReference struct {
 type ThanosQuerierReference struct {
 	// Url of the ThanosQuerier to proxy to.
 	//
-	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:MinLength:=1
-	Url string `json:"url"`
+	// +kubebuilder:validation:Optional
+	Url string `json:"url,omitempty"`
+}
+
+// Perses is used to configure a reference to a perses service that should be used
+// by the monitoring console plugin.
+//
+// +structType=atomic
+type PersesReference struct {
+	// Name of the Perses Service to proxy to.
+	//
+	// +kubebuilder:validation:Optional
```

Comment:
We should have a enabled attribute here. If so then we can reconcile the Perses instance, name and namespace are optional in the case the plugin does not point to the default perses installation

Reply:
Added "enabled" attribute to PersesReference, ACMReference, and IncidentsReference structs